cider/exec/template/Execute.h

```diff
@@ -669,6 +672,8 @@ class Executor {
   }
 
   std::unique_ptr<CgenState> cgen_state_;
+  CiderStringHasher* stringHasher_;
+  mutable std::mutex str_hasher_mutex_;
```

Comment:
why this std::mutex variable is mutable?

Reply:
same as line 705.